docs/content/Box.md

```diff
@@ -4,33 +4,75 @@ title: Box
 
-## Default example
-
-```jsx live live
-<Box>
-  Box can be used to create both{' '}
-  <Box as="span" color="text.inverse" bg="bg.successInverse">
-    inline
-  </Box>{' '}
-  and
-  <Box color="text.inverse" bg="bg.dangerInverse">
-    block-level elements,
+## Examples
+
+### Default
+
+```jsx live
+<Box color="text.secondary" bg="bg.tertiary" p={3}>
+  Hello
+</Box>
+```
+
+### Border on all sides
+
+```jsx live
+<Box borderColor="border.primary" borderWidth={1} borderStyle="solid" p={3}>
+  Hello
+</Box>
+```
+
+### Border on one side
+
+```jsx live
+<Box borderColor="border.primary" borderBottomWidth={1} borderBottomStyle="solid" pb={3}>
+  Hello
+</Box>
+```
+
+### Flexbox
+
+Use `Box` to create [flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox) layouts.
+
+```jsx live
+<Box display="flex">
+  <Box p={3} borderColor="border.primary" borderWidth={1} borderStyle="solid">
+    1
+  </Box>
+  <Box flexGrow={1} p={3} borderColor="border.primary" borderWidth={1} borderStyle="solid">
+    2
+  </Box>
+  <Box p={3} borderColor="border.primary" borderWidth={1} borderStyle="solid">
+    3
+  </Box>
+</Box>
+```
+
+### Grid
+
+Use `Box` to create [grid](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids) layouts.
+
+```jsx live
+<Box display="grid" gridTemplateColumns="1fr 1fr" gridGap={3}>
+  <Box p={3} borderColor="border.primary" borderWidth={1} borderStyle="solid">
+    1
   </Box>
-  <Box color="text.inverse" bg="bg.warningInverse" width={[1, 1, 1 / 2]}>
-    elements with fixed or responsive width and height,
+  <Box p={3} borderColor="border.primary" borderWidth={1} borderStyle="solid">
+    2
   </Box>
-  <Box color="text.inverse" bg="bg.infoInverse" p={4} mt={2}>
-    and more!
+  <Box p={3} borderColor="border.primary" borderWidth={1} borderStyle="solid">
+    3
   </Box>
 </Box>
 ```
 
 ## System props
 
-Box components get the `COMMON`, `LAYOUT`, and `FLEX` categories of system props. Read our [System Props](/system-props) doc page for a full list of available props.
+Box components may receive system props of any category. Read our [System Props](/system-props) doc page for a full list of available props.
 
 ## Component props
 
-| Prop name | Type   | Default | Description                         |
-| :-------- | :----- | :-----: | :---------------------------------- |
-| as        | String |  `div`  | sets the HTML tag for the component |
+| Prop | Type                |                                 Default                                  | Description |
+| :--- | :------------------ | :----------------------------------------------------------------------: | :---------- |
+| `as` |                     | [`"div"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div) |             |
+| `sx` | `SystemStyleObject` |                                    —                                     |             |
```

Comment:
Let's consolidate the `System props` and `Component props` sections into a `Props` section. Maybe something like this: 
```suggestion
| `sx` | `SystemStyleObject` |                                    —                                     |             |
`Box` also accepts all [styled system props].
```